scene/gui/line_edit.cpp

```diff
@@ -1713,6 +1713,12 @@ void LineEdit::set_caret_column(int p_column) {
 	} else if (MAX(primary_caret_offset.x, primary_caret_offset.y) >= ofs_max) {
 		scroll_offset += ofs_max - MAX(primary_caret_offset.x, primary_caret_offset.y);
 	}
+
+	// Scroll to show as much text as possible
```

Comment:
```suggestion
	// Scroll to show as much text as possible.
```
:P

Reply:
Well, I don't know how to amend a commit after the PR has been merged. Not even sure I've permission.

Reply:
It's not possible, you'd need to open a new PR.
Though such random typo fixes are usually done in batches, so it's just something to remember for later.

Reply:
Cool, I'll keep it in mind. Thanks for the heads up!